sample_creation.py

```diff
@@ -15,17 +15,29 @@
 from tqdm import tqdm
```

Comment:
Should we remove this script and get it back in old versions of GDL when ready to reimplement? #223 
Not sure it's relevant to keep it and maintain it for now.

Reply:
What was use for ?

Reply:
It was used to received single-band imagery as input. This will be tackled in the coming month or so with issue #223 and #222 . I think we can remove it if @victorlazio109 is okay with that. Otherwise, we can wait and remove it when a different solution is merged.